src/actions/Entities/SensorActions.js

```diff
@@ -177,7 +177,7 @@ const createNew = () => (dispatch, getState) => {
     if (existingLocation) {
       newLocation = UIDatabase.update('Location', {
         id: existingLocation.id,
-        description: sensor.name,
+        description: location.name,
```

Comment:
Minor nit for readability (equivalent as `location.description = sensor.name` on `SensorActions:170`).